packages/react/src/components/Tag/Tag.js

```diff
@@ -46,8 +47,13 @@ const Tag = ({
     [`${prefix}--tag--${type}`]: type,
   });
   const handleClose = event => {
-    event.stopPropagation();
-    onClose(event);
+    if (onClose) {
+      event.stopPropagation();
+      onClose(event);
+    }
+    if (onClick && !onClose) {
```

Comment:
Wondering if `onClick` is necessary here? Should this not just bubble up to the div where the `onClick` handler is located and where the `...other` props are spread?

Reply:
Ah right, updated it now 👍